src/main/kotlin/com/criptext/mail/scenes/mailbox/MailboxSceneController.kt

```diff
@@ -783,6 +825,13 @@ class MailboxSceneController(private val scene: MailboxScene,
         scene.showDialogLabelsChooser(LabelDataHandler(this))
     }
 
+    private fun checkRecommendBackup(threadSize: Int){
+        if(threadSize > 10
+                && !storage.getBool(KeyValueStorage.StringKey.HasBeenAskedRecommendBackup, false)){
```

Comment:
This doesn't work for me. The popup was shown even thought I had backup enabled.

Reply:
Also, the validation is for each account